client/state/themes/actions.js

```diff
@@ -521,8 +528,8 @@ export function tryAndCustomize( themeId, siteId ) {
 export function installAndTryAndCustomizeTheme( themeId, siteId ) {
 	return ( dispatch ) => {
 		return dispatch( installTheme( themeId, siteId ) )
-			.then( () => {
-				dispatch( tryAndCustomizeTheme( themeId, siteId ) );
+			.then( ( id ) => {
+				dispatch( tryAndCustomizeTheme( id || themeId, siteId ) );
```

Comment:
This sort of implicit fallback is a pattern that I'd rather avoid -- I think it'll make debugging later quite a bit harder. We're only falling back to `themeId` if `id` is falsey, but even now when only seeing the diff, I'm not sure if there are any circumstances when it'll actually be falsey.

So can we just drop the `|| themeId` here? If we take care of the special case with the suffix inside `installTheme`, it'd be great to contain it there and not spill the special casing outside of it.